PlayCover/en.lproj/Localizable.strings

```diff
@@ -94,7 +94,8 @@
 "alert.wrongFileType.title" = "Wrong file type!";
 "alert.wrongFileType.subtitle" = "PlayCover can only install valid .ipa files.";
 "alert.install.injectPlayTools" = "Inject PlayTools";
-"alert.install.injectPlayToolsInApp"= "Do you want to inject PlayTools into this application?";
+"alert.install.injectPlayToolsQuestion" = "Do you want to inject PlayTools into this application?";
+"alert.install.playToolsInformative" = "PlayTools is a tool that allows you to bind keys to screen touches, custom display resolutions, and more, but may break certain apps";
```

Comment:
```suggestion
"alert.install.playToolsInformative" = "PlayTools is a tool that allows you to bind keys to screen touches, custom display resolutions, and more, but it may break certain apps";
```